source/install/cloud-changelog.md

```diff
@@ -4,12 +4,49 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-06-29](#release-2022-06-29)
 - [Release 2022-06-22](#release-2022-06-22)
 - [Release 2022-06-15](#release-2022-06-15)
 - [Release 2022-06-14](#release-2022-06-14)
 - [Release 2022-05-26](#release-2022-05-26)
 - [Release 2022-05-12](#release-2022-05-12)
-- [Release 2022-04-28](#release-2022-04-28)
+
+## Release 2022-06-29
+
+### Highlights
+
+#### Insights (Beta)
+ - Added workplace insights consisting of usage and behaviour data, which helps Enterprises further increase productivity of their employees through Mattermost functionality.
+
+### Improvements
+
+#### User Interface (UI)
+- Added support for syntax highlighting for 1C:Enterprise (BSL) language.
+- Added support for WebP image types for emojis and image file uploads.
```

Comment:
Similar comment here; looks like WebP is a modern image format for superior lossless and lossy compression for web images. That's a new one for me! :) If we feel it may be new for others as well, consider elaborating a bit on the value.

Reply:
I'll add @hmhealey to confirm the benefit/value.